database/migrations/2024_03_20_000000_modify_contents_data_type.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class () extends Migration {
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::table('pages', function (Blueprint $table) {
+            $table->json('contents')->change();
+        });
+    }
+
+    /**
+     * Reverse the migrations.
+     */
+    public function down(): void
+    {
+        Schema::dropIfExists('pages');
```

Comment:
This isn't the reverse of the `up` operation.  Please correct it to just revert the change to the 'contents' column

Reply:
Yep sorry that shouldn't be there